plugins/spf.js

```diff
@@ -42,6 +44,9 @@ exports.load_config = function () {
             '-deny_relay.mfrom_softfail',
             '-deny_relay.mfrom_fail',
             '-deny_relay.mfrom_permerror',
+
+            '-bypass.relaying',
+            '-bypass.auth',
```

Comment:
Instead of 'bypass', we have an established pattern of using the term 'skip' instead. Consistency is preferable.